testgrid/conformance/upload_e2e.py

```diff
@@ -210,13 +230,25 @@ def parse_args(cli_args=None):
     parser.add_argument(
         '--metadata',
         help='dictionary of additional key-value pairs that can be displayed to the user.',
+        required=False,
         default=str(),
     )
+    parser.add_argument(
+        '--key-file',
+        help='path to GCP service account key file, which will be activated with'
```

Comment:
Do they have to pass the key only once and then will they always stay activated? Or do they have to pass the key with each upload? It isnt clear from this decsription.

Reply:
It's activating before upload. Typically it should activate every time because the environment isn't persistent. If this were run on a workstation it would remain as the activated account though.


Key activation = "login to gcloud with this key and make it the current account"